### ISSUE TITLE
Smart recommendations of the services to look into

### ISSUE THREAD

**Issue by doobneek1** [streetlives/yourpeer.nyc#247]
Within the user's session gather information on the types of services and locations they are visiting and suggest what other services to check out in a new sorting method.

**Comment by adambard1** [comment]
Yes it would be great to have a smart match 'You may also like' feature
Would need some design to start.
